clientd3d/animate.c

```diff
@@ -27,7 +27,7 @@
 
 #include "client.h"
 
-#define ANIMATE_INTERVAL 165       // ms between background animation updates
+#define ANIMATE_INTERVAL 40       // ms between background animation updates
```

Comment:
40ms is an animation interval I pulled from `firewand.kod`, but I'm out of my depth here. Is this the right move to eliminate the difference in background animations or should it be a number close to 0?

Reply:
The results of 40ms vs. 0ms seem obvious, but here's footage for comparison (when the dialog is present) for anyone following along:

`ANIMATE_INTERVAL 40`

https://user-images.githubusercontent.com/467443/228704161-6f1a45a3-7dc9-434a-a2a2-924886d20cba.mp4

`ANIMATE_INTERVAL 0`

https://user-images.githubusercontent.com/467443/228704244-04f19ce8-a036-47ce-8d65-6a2b81ed43b7.mp4